benchmarks/upload_local.py

```diff
@@ -0,0 +1,42 @@
+import numpy as np
+import logging
+
+logging.basicConfig(level=logging.ERROR)
+
+
+import hub
+from benchmark_utils import timer, parametrize
+from tfds_utils import datasets, download, DATASETS_DIR, get_hub_ds_path
+import posixpath
+
+
+username = open("username.txt", "r").read()
+
+
+@parametrize(dataset=datasets)
```

Comment:
so this is a custom parametrize? if it is, could you rename it? because we have a lot of these custom parametrizes, and if they all were named parametrize it would be frightening